engine/src/flutter/shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -2519,6 +2519,10 @@ - (void)removeEnableFlutterTextInputViewAccessibilityTimer {
   return _activeView;
 }
 
+- (void)reset {
+  [self hideTextInput];
```

Comment:
@justinmc i remember there was an issue with hot reload where the native edit menu was not dismissed after hot reload? is it still an issue? if so, this is a good place to dismiss the menu.

Reply:
Just to confirm, the native edit menu is the Copy/Cut/Paste context menu you see when you select text? If so, I was able to reproduce this not getting dismissed on hot restart on latest master, but this PR also fixes that bug :)

Reply:
Yes that's what I meant. That's great! 

Worth writing a comment there explaining what you are doing.

Reply:
Added a comment!